src/wxflow/configuration.py

```diff
@@ -96,7 +98,12 @@ def print_config(self, files: Union[str, bytes, list]) -> None:
     def _get_script_env(cls, scripts: List) -> Dict[str, Any]:
         default_env = cls._get_shell_env([])
         and_script_env = cls._get_shell_env(scripts)
-        vars_just_in_script = set(and_script_env) - set(default_env)
+        keys_in_scripts = set()
+        regex_pattern = 'export\\s+\\b(' + '|'.join(map(re.escape, default_env.keys())) + ')(?==)'
+        for script in scripts:
+            result = subprocess.run(['grep', '-o', '-P', regex_pattern, script], stdout=subprocess.PIPE)
+            keys_in_scripts.update(result.stdout.decode().split())
```

Comment:
Can this be explained a bit?
The task is simple here;  get a dictionary of key-value pairs set by a shell script.

Reply:
The old approach was faulty, this one (not intended to be used) resorted to using a regular expression to fish out any environment variable that is in the user spaces that is also in a config file.  I don't like this so I'm thinking of another approach entirely to avoid chicken-before-egg dilemma that this will not solve.

Reply:
Yes and then just check if the key is in both!